data/ShipEngineDemoData.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<entity-facade-xml type="demo">
+    <mantle.shipment.ShipmentBoxType shipmentBoxTypeId="USPS_SmFlat" description="USPS Small Flat Rate Box" dimensionUomId="LEN_in" boxLength="8.69" boxWidth="5.44" boxHeight="1.75" gatewayBoxId="USPS_SmallFlatRateBox"/>
+    <mantle.shipment.ShipmentBoxType shipmentBoxTypeId="USPS_MdFlat1" description="USPS Medium Flat Rate Box 1" dimensionUomId="LEN_in" boxLength="11.25" boxWidth="8.75" boxHeight="6" gatewayBoxId="USPS_MediumFlatRateBox1"/>
+    <mantle.shipment.ShipmentBoxType shipmentBoxTypeId="USPS_LgFlat" description="USPS Large Flat Rate Box" dimensionUomId="LEN_in" boxLength="12.25" boxWidth="12.25" boxHeight="6" gatewayBoxId="USPS_LargeFlatRateBox"/>
+
+    <mantle.shipment.carrier.ShippingGatewayConfig shippingGatewayConfigId="SHIPENGINE_DEMO"
+                                                   shippingGatewayTypeEnumId="ShGtwyShipEngine" description="ShipEngine API Demo"
+                                                   getRateServiceName="mantle.ShipEngine.ShipEngineServices.get#ShippingRate"
+                                                   requestLabelsServiceName="mantle.ShipEngineServices.post#CreateLabel">
+
+        <!--  set your ShipEngine Account IDs for carriers with records like these  -->
+        <carriers carrierPartyId="UPS" gatewayAccountId="se-2084091"/>
+        <carriers carrierPartyId="USPS" gatewayAccountId="se-2084090"/>
+
+        <!--  set your ShipEngine API token (test or live) with a record like this  -->
+        <options optionEnumId="SgoApiToken" optionValue=""/>
+        <options optionEnumId="SgoLabelType" optionValue="PDF_4x6"/>
+
+        <!--  use SCopyhippingGatewayBoxType to override default gatewayBoxId on ShipmentBoxType  -->
+        <boxTypes shipmentBoxTypeId="USPS_SmFlat" gatewayBoxId="USPS_SmallFlatRateBox"/>
+
+        <!--  use ShippingGatewayMethod to override default gatewayServiceCode on CarrierShipmentMethod  -->
+        <methods carrierPartyId="USPS" shipmentMethodEnumId="ShMthGround" gatewayServiceCode="usps_first_class_mail"/>\
```

Comment:
Make sure that gatewayServiceCode mapping here is correct for USPS Ground.

Reply:
Changed gatewayServiceCode